src/3p-frame.js

```diff
@@ -65,7 +65,7 @@ function getFrameAttributes(parentWindow, element, opt_type) {
     locationHref = parentWindow.parent.location.href;
   }
   attributes._context = {
-    referrer: viewer.getReferrerUrl(),
+    referrer: viewer.getUnconfirmedReferrerUrl(),
```

Comment:
Do we really need to change this? In theory this is a relatively benign thing to spoof referrer here and in dynamic CSS classes. The main cases where it matters, e.g. FCF, we used confirmed referrer URL.

The problem is that here, it'd be very undesirable to make this code async for two reasons:

1. It's meant to be sync since it's called in the mutate phase of DOM construction
2. It's more or less guaranteed that this construction will be happening before viewer's comms link is established. Sometimes significantly so. In my tests, this was the case 100% of the time.